src/Donations/PaymentMethods/NewPaypal.tsx

```diff
@@ -65,10 +75,16 @@ function NewPaypal({
       payDonationFunction("paypal", "paypal", data);
     });
   }
-
+  function onApproveSubscription(data, actions) {
+    // This function shows a transaction success message to your buyer.
+    data = {
+      ...data,
+      type: "sdk",
+    };
+    payDonationFunction("paypal", "paypal", data);
+  }
   const onError = (data) => {
     setPaymentError(`Your order failed due to some error.`);
```

Comment:
Translate.

Reply:
Come on - please do not just fix this, fix them all! :-)

<img width="455" alt="Bildschirmfoto 2022-01-04 um 14 06 29" src="https://user-images.githubusercontent.com/1532418/148063607-bfbb9d7a-407d-4a13-929f-f36888a99ef7.png">